stylix/testbed/images.nix

```diff
@@ -9,7 +9,9 @@
   };
 
   light = fetchurl {
-    name = "three-bicycles.jpg";
+    # Includes a space to make sure the path is properly quoted via
+    # `lib.escapeShellArg` when used as a shell argument.
+    name = "three bicycles.jpg";
```

Comment:
Not sure if this space actually remains in the path, or if it gets transformed back to a `-`. If it does then we'd need to add another derivation which places the file inside a directory and returns that path.